client/src/main/java/org/apache/cloudstack/ServerDaemon.java

```diff
@@ -302,9 +301,31 @@ private RequestLog createRequestLog() {
         log.setAppend(true);
         log.setLogTimeZone("GMT");
         log.setLogLatency(true);
+        createRotateFile(logPath);
         return log;
     }
 
+    private void createRotateFile(File logPath) {
+        String rotatefile = "/etc/logrotate/access";
```

Comment:
should this not be /etc/logrotate.d/ ?

Reply:
```suggestion
        String rotatefile = "/etc/logrotate.d/access";
```

Reply:
thanks @kriegsmanj care to do some testing on this work in progres?